### PR TITLE
change issue name on title, add issue table, small fixes

### PR DESCRIPTION
**Pull request by andrewRv**

### CODE REVIEW

File: client/apps/city_issues/views.py [hunk 1]
```diff
@@ -6,53 +6,73 @@
...
+from django.views.generic.base import TemplateView, View
 from django.contrib import messages
 from django.core import serializers
 from django.http import JsonResponse
 from django.shortcuts import redirect, render
 from django.urls import reverse
 from django.utils.timezone import make_aware
 
-from city_issues.models import Attachments, Issues
-from city_issues.forms.forms import EditIssue, IssueFilter, IssueForm
```
Comment (KostSvitlana) [inline client/apps/city_issues/views.py:18]: Why you delete `IssueFilter`? Now page with map open with error "global name 'IssueFilter' is not defined"

File: client/apps/city_issues/views.py [hunk 2]
```diff
@@ -6,53 +6,73 @@
...
 class HomePageView(TemplateView):
     """Home page"""
     template_name = "home_page.html"
 
 
+class UserProfileView(View):
+
+    def get(self, request, user_id):
+        user = User.objects.get(id=user_id)
+        user_issues = IssueHistory.objects.filter(user=user).select_related('issue')
```
Comment (KostSvitlana) [inline client/apps/city_issues/views.py:30]: Why do you select issue through `IssueHistory`, I think it return wrong data and issue duplicated.
I tried that way `user_issues = Issues.objects.filter(user=user).select_related('user')`

File: client/apps/city_issues/templates/user/user.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                </tfoot>
+                <tbody>
+                {% for isssue in user_issues %}
+                    <tr>
+                        <td>{{ forloop.counter }} </td>
+                        <td>{{ isssue.issue.title }}</td>
+                        <td>{{ isssue.issue.category.category }}</td>
+                        <td>{{ isssue.issue.status }}</td>
+                        <td>{{ isssue.issue.open_date }}</td>
+                        <td>{{ isssue.issue.close_date }}</td>
```
Comment (KostSvitlana) [inline client/apps/city_issues/templates/user/user.html:97]: If you change query then here will be so
`{% for issue in user_issues %}
    <tr>   <td>{{ forloop.counter }} </td>
    <td>{{ issue.title }}</td> ... `

File: client/basic/urls.py [hunk 4]
```diff
@@ -16,11 +16,13 @@
 from django.contrib.auth import views as auth_views
 from django.conf import settings
 from django.conf.urls import include, url
+from django.conf.urls import include, url
```
Comment (KostSvitlana) [inline client/basic/urls.py:19]: Here dublicate import

File: client/apps/city_issues/views.py (hunk 2 above)
Comment (andrewRv) [reply client/apps/city_issues/views.py:30]: You're right, I confused the models

**Pull request closed without merging**